posthog/api/test/test_decide.py

```diff
@@ -803,6 +803,76 @@ def test_feature_flags(self, *args):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.json()["featureFlags"], ["default-flag"])
 
+    def test_feature_flags_across_multiple_environments(self, *args):
```

Comment:
Just to make sure I understand this test. Is an "environment" in this context a "team"? So the test is ensuring that calling `/decide` with the second team's token will be able to access feature flags's created in the first team?

Can a feature flag with the same name be in both environments? If so, what happens then?

Reply:
Yes and yes! `Team` = environment, `Project` = project. As for key uniqueness, flags have to be unique at the _project_ level – this is already enforced